Server/Content/.gitignore

```diff
@@ -0,0 +1,9 @@
+Packages
```

Comment:
You can remove this file, because there is already a `.gitignore` in the root folder